packages/react-docs/RELEASE-NOTES.md

```diff
@@ -3,6 +3,125 @@ id: Release notes
+- [@patternfly/react-core@4.147.0](https://www.npmjs.com/package/@patternfly/react-core/v/4.147.0)
+- [@patternfly/react-icons@4.11.8](https://www.npmjs.com/package/@patternfly/react-icons/v/4.11.8)
+- [@patternfly/react-inline-edit-extension@4.7.44](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.7.44)
+- [@patternfly/react-log-viewer@4.4.0](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/4.4.0)
+- [@patternfly/react-styles@4.11.8](https://www.npmjs.com/package/@patternfly/react-styles/v/4.11.8)
+- [@patternfly/react-table@4.29.37](https://www.npmjs.com/package/@patternfly/react-table/v/4.29.37)
+- [@patternfly/react-tokens@4.12.9](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.12.9)
+- [@patternfly/react-topology@4.9.42](https://www.npmjs.com/package/@patternfly/react-topology/v/4.9.42)
+- [@patternfly/react-virtualized-extension@4.9.11](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.9.11)
+
+### Components
+- **Accordion:**
+  - Added support for bordered, display large and multiple body content ([#6085](https://github.com/patternfly/patternfly-react/pull/6085))
+- **Alert Group:**
+  - Made dynamic alerts more accessible ([#5946](https://github.com/patternfly/patternflg-react/pull/5946))
+- **Avatar:**
+  - Added border prop ([#6040](https://github.com/patternfly/patternfly-react/pull/6040))
+- **Button:**
+  - Updated to allow router link component ([#6061](https://github.com/patternfly/patternfly-react/pull/6061))
+- **Checkbox:**
+  - Fixed warning about ouiaSafe property ([#6055](https://github.com/patternfly/patternfly-react/pull/6055))
+  - Added OUIA props ([#5949](https://github.com/patternfly/patternfly-react/pull/5949))
+- **Code editor:**
+  - Added support for more languages ([#6006](https://github.com/patternfly/patternfly-react/pull/6006))
+  - Added showEditor prop ([#6022](https://github.com/patternfly/patternfly-react/pull/6022))
+  - Updated to allow passing options and overrideServices ([#6035](https://github.com/patternfly/patternfly-react/pull/6035))
+  - Updated to show toolbar if there are custom controls ([#6025](https://github.com/patternfly/patternfly-react/pull/6025))
+- **Context selector:**
+  - Updated to hide internal prop ([#5986](https://github.com/patternfly/patternfly-react/pull/5986))
+- **Date picker:**
+  - Changed default parse format to y-m-d from y(-m)?(-d)? ([#6039](https://github.com/patternfly/patternfly-react/pull/6039))
+  - Cleared parse error on type ([#6066](https://github.com/patternfly/patternfly-react/pull/6066))
+- **Dual list selector:**
+  - Fixed button order ([#5983](https://github.com/patternfly/patternfly-react/pull/5983))
+- **Expandable section:**
+  - Added props to support disclosure variation ([#6078](https://github.com/patternfly/patternfly-react/pull/6078))
+- **Form:**
+  - Added form group label info ([#6053](https://github.com/patternfly/patternfly-react/pull/6053))
```

Comment:
Do we also want to call out the password strength demo added here?
`feat(Form): add form group label info & docs(Form): add password strength demo #6053`

Reply:
Nevermind - see this listed under the "Demos" section.